backend/copr_backend/background_worker_build.py

```diff
@@ -381,7 +382,25 @@ def _proctitle(self, text):
         self.log.debug("setting title: %s", text)
         self.setproctitle(text)
 
+    def _check_build_interrupted(self):
+        """
+        Should we interrupet a running worker?
+        """
+        return (self._cancel_task_check_request()
+                or self._check_failed_resalloc_ticket())
```

Comment:
Maybe a nitpick; if the ticket is FAILED, there's no point in trying to cancel the remote process ... so I'd at least reverse these checks.

Reply:
It is somewhat difficult to verify this behaves as expected :-) because the return value is cached.